cmd/pull.go

```diff
@@ -43,14 +45,18 @@ var pullCmd = &cobra.Command{
 		//build the module and name query paramaters
 		query := url.Values{}
 
+		if noModule && module == "" && page == "" {
```

Comment:
seems like the `&& page == ""` is not necessary, you should be able to use `nomodule` in conjunction with `page` to ensure that you are getting a page that is not in a module.

For instance, if you've got 2 pages, both called "Foo", but one is in a module and one is not, then using `nomodule` and `page=Foo` would ensure that you only get back 1 result, instead of 2 results which you would get back if you did just `page=Foo` without `nomodule`

Reply:
@zachelrath Yeah, I think I lost sight of this scenario as I kept thinking there would only be a singular page with a particular name rather than multiple as you described above.